ftd/src/executor/styles.rs

```diff
@@ -823,22 +1032,365 @@ impl Background {
         }
     }
 
-    pub fn to_image_css_string(&self) -> String {
+    pub fn to_image_src_css_string(&self) -> String {
+        match self {
+            Background::Solid(_) => ftd::interpreter2::FTD_IGNORE_KEY.to_string(),
+            Background::Image(i) => i.to_image_src_css_string(),
+        }
```

Comment:
```suggestion
        match self {
            ftd::executor::Background::Image(i) => i.to_image_src_css_string(),
            _ => ftd::interpreter2::FTD_IGNORE_KEY.to_string(),
        }
```

This helps to understand that only the `Image` variant is of interest.